### PR TITLE
cdist

### LINKED ISSUE #21894: cdist

### PR DESCRIPTION
**Pull request by teckno**
closes #21894

### CODE REVIEW

**Comment by ivy-leaves** [comment]
If you are working on an open task, please **edit** the PR description to link to the issue you've created.

For more information, please check [ToDo List Issues Guide](https://unify.ai/docs/ivy/overview/contributing/the_basics.html#todo-list-issues).

Thank you :hugs:

File: ivy/functional/frontends/torch/miscellaneous_ops.py [hunk 1]
```diff
@@ -454,3 +454,8 @@ def corrcoef(input):
             f" input with {ivy.shape(input)} dimansions"
         )
     return ivy.corrcoef(input, y=None, rowvar=True)
+
+@to_ivy_arrays_and_back
+def cdist(x1, x2, p=2.0, compute_mode='use_mm_for_euclid_dist_if_necessary'):
+    return ivy.cdist(x1, x2, p=p, compute_mode=compute_mode)
```
Comment (hirwa-nshuti) [inline ivy/functional/frontends/torch/miscellaneous_ops.py:460]: This function isn't present in the Ivy API

**Review by hirwa-nshuti — CHANGES_REQUESTED** [review]
Hey @teckno 

Check the comments below🙂
Thanks

**Review by hirwa-nshuti — CHANGES_REQUESTED** [review]
Would you also resolve the merge conflicts for CI checks🤞

**Comment by Sarvesh-Kesharwani** [comment]
ChibuezeOnejeme

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

**Pull request closed without merging**